src/components/mainMenuButton.js

```diff
@@ -0,0 +1,11 @@
+const MainMenuButton = (className,innerText,callback) => {
+    const button = document.createElement('button');
+    button.classList.add(className)
```

Comment:
To do tego komentarza wyżek:
```suggestion
const MainMenuButton = (innerText,callback) => {
    const button = document.createElement('button');
    button.classList.add('mainMenu__button');
```